src/ai/ai.service.ts

```
@@ -0,0 +1,173 @@
+    @Inject(forwardRef(() => GameService))
+    private readonly gameService: GameService,
+    private readonly geminiService: GeminiService,
+  ) {}
+
+  /**
+   * Creates a Stockfish engine instance (WASM via `stockfish` package).
+   * Each request uses a short-lived engine for simplicity & isolation.
+   */
+  private createEngine(): any {
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const Stockfish = require('stockfish');
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)